vm/vm.go

```diff
@@ -378,6 +400,52 @@ func walkBinary(ctx expr.EvalContext, node *expr.BinaryNode) (value.Value, bool)
 				u.Errorf("at?%T  %v  coerce?%v bt? %T     %v", at, at.Value(), at.CanCoerce(stringRv), br, br)
 			}
 		}
+	case value.SliceValue:
+		switch node.Operator.T {
+		case lex.TokenContains:
+			switch br.(type) {
+			case nil, value.NilValue:
+				return nil, false
+			case value.StringValue:
+				// [x,y,z] contains str
+				for _, val := range at.Val() {
+					//u.Infof("str contains? %v %v", val.Value(), br.Value())
+					if eq, _ := value.Equal(val, br); eq {
+						return value.BoolValueTrue, true
+					}
+				}
+				return value.BoolValueFalse, true
+			case value.IntValue:
```

Comment:
I wouldn't even bother with non-string types. The deprecated system we're replacing only applies this operator to strings and it's explicitly called "*str* contains"

If it were a simple equality check I'd be fine extending it, but since it's a recursive contains operation applying it to other types seems ... weird.